static/index.html

```diff
@@ -1,52 +1,71 @@
-<!DOCTYPE html>
-<html ng-app="cfdeck">
-	<head>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="assets/style.css">
+  </head>
+  <style>
+    /* Move down content because we have a fixed navbar that is 50px tall */
```

Comment:
more of a question: how will this padding impact the content on a mobile device.